packages/shared/ReactSymbols.js

```diff
@@ -34,6 +34,7 @@ export const REACT_MEMO_TYPE: symbol = Symbol.for('react.memo');
 export const REACT_LAZY_TYPE: symbol = Symbol.for('react.lazy');
 export const REACT_SCOPE_TYPE: symbol = Symbol.for('react.scope');
 export const REACT_OFFSCREEN_TYPE: symbol = Symbol.for('react.offscreen');
```

Comment:
Is this still used anywhere or can this be deleted now?

Reply:
Basically the idea is that React internally only refers to Offscreen using the tag. It should never have to look at `.type` or `.elementType`. The `createFiberFromOffscreen` should exclude it since it's not needed for the `elementType`. It can be `null`. `elementType` is only used for reconciliation against new elements but since Offscreen is only created internally it should never need to be reconciled. And so it should be impossible to create this symbol an all the cases that try to match it.